lib/src/Logger.cpp

```diff
@@ -277,6 +286,8 @@ Logger::selectCorrectLogLevel(int chosenPriority) const
 void
 Logger::log(int priority, const char* mess, ...)
 {
+    printAlsoToConsoleIfRequired(mess);
```

Comment:
I'm wondering if `printAlsoToConsoleIfRequired` is expected to be always executed, I think it should be moved near call to `priv_log` otherwise we will get messages logged on console but not written to file.

Reply:
@brushtyler i think this should be expected behaviour instead... when debugging you want to see those messages and we do not have logging priority. If such feature is needed then we could use a consolelogger.